### DISCUSSION TITLE
Running notify with sudo gives command not found

### DISCUSSION THREAD

**Discussion by njmulsqb** [projectdiscovery/notify#104] [Q&A] [answered]
Hi,

I am running notify's latest version (just fetched today), if I run notify simply using `notify` it works well but if I type `sudo notify` it says command is not found!

I have other project discovery tools installed as well e.g nuclei, httpx etc and they dont behave this way they works well both with and without sudo.

**Comment by njmulsqb** [comment]
Is there anyone?

**Reply by ehsandeep** [reply, marked as answer]
@njmulsqb this is because `sudo` and installing user is not same, it's an issue with your setup, additionally, you can try installing notify with `sudo` and see if it works after that.
